src/RecoveryServices/RecoveryServices/help/Set-AzRecoveryServicesBackupProtectionPolicy.md

```diff
@@ -55,14 +57,14 @@ $RetPol.WeeklySchedule.DurationCountInWeeks = 365
 $vault = Get-AzRecoveryServicesVault -ResourceGroupName "azurefiles" -Name "azurefilesvault"
 $Pol= Get-AzRecoveryServicesBackupProtectionPolicy -Name "TestPolicy" -VaultId $vault.ID
 $Pol.SnapshotRetentionInDays=5
-Set-AzRecoveryServicesBackupProtectionPolicy -Policy $Pol -SchedulePolicy $SchPol -RetentionPolicy $RetPol
+Set-AzRecoveryServicesBackupProtectionPolicy -Policy $Pol -SchedulePolicy $SchPol -RetentionPolicy $RetPol -AzureBackupSnapshotRGName "snapshotResourceGroupPrefix" -AzureBackupSnapshotRGName "snapshotResourceGroupSuffix"
```

Comment:
"Az.RecoveryServices","Set-AzRecoveryServicesBackupProtectionPolicy","1","19","Invalid_Parameter_Name","5011","1","Set-AzRecoveryServicesBackupProtectionPolicy -AzureBackupSnapshotRGName is not a valid parameter name.","-AzureBackupSnapshotRGName","Check validity of the parameter -AzureBackupSnapshotRGName."